src/frontend/web_application/src/components/ContactDetails/components/FormButton/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Button from '../../../Button';
+
+import './style.scss';
+
+class FormButton extends Component {
+  static propTypes = {
+    __: PropTypes.func.isRequired,
+    obj: PropTypes.shape({}).isRequired,
```

Comment:
`obj` is way too much abstract imo, it should give a hint to "stringify" or a render method in there.

But actually it's a `node` in react terms. And Because FormButton is more like a decorator, so you can simply describe a children property:

`children: PropTypes.node.isRequired`